spec/features/admin/evaluations_spec.rb

```diff
@@ -10,38 +10,110 @@
   end
 
   describe 'Filters' do
+    let!(:user1) { create(:user,
+                          active: true,
+                          company: admin_user.company) }
+
+    let!(:user2) { create(:user,
+                          active: true,
+                          company: admin_user.company) }
+
+    let!(:evaluation1) { create(:evaluation,
+                                :english,
+                                evaluated: user1,
+                                evaluator: user2,
+                                evaluation_date: 3.week.ago,
+                                created_at: 5.month.ago,
+                                company: admin_user.company) }
```

Comment:
```suggestion
    let!(:evaluation1) do
      create(:evaluation,
        :english,
        evaluated: user1,
        evaluator: user2,
        evaluation_date: 3.week.ago,
        created_at: 5.month.ago,
        company: admin_user.company
      )
    end
```
You should use `do ... end` instead of `{}` when using a multi line block :nail_care: 

I'd say you could also make the tabs just expand once inside the `create` methods instead of aligning them with `:evaluation`

Reply:
I used some example in the code that was using multi-line with `{}` and the editor was saying to use `do ... end` but I didn't know if was more precise and I leave it. Thanks for the suggestion and about the indentation, the editor warn about it:
![image](https://user-images.githubusercontent.com/30053543/170069991-db43e179-3967-47a4-a52a-9719efbf5cae.png)

And if I do this way It works:
![image](https://user-images.githubusercontent.com/30053543/170070103-43f3c642-3ffc-433d-9782-b7757187d10f.png)

But I prefer the first one, because is simple to search locations that are using `create(:evaluation` this way.